.gitignore

```diff
@@ -3,6 +3,6 @@
 .idea/
 sites/**
 !sites/default
-sites/default/sqlconfig
+sites/default/sqlconfig.php
```

Comment:
@muarachmann  the file `sqlconfig.php`  does not exist, I think you meant `sqlconf.php` as shown in '1' below
![image](https://user-images.githubusercontent.com/22508583/66551948-5e0f3500-eb40-11e9-9661-fac5fbdc2a38.png)

- for development purposes, users who had already installed this system will need to run  `git rm --cached .\sites\[path_to_active_site_name]\sqlconf.php`,  for example,  mine is (git rm --cached .\sites\default\sqlconf.php) before git can start ignoring the file.